openapi/openapi.yml

```diff
@@ -1360,6 +1360,11 @@ paths:
               properties:
                 name:
                   type: string
+                tags:
+                  type: array
```

Comment:
We should have a `description` here. I'm not sure why a `description` is missing from the `name` property, we have it on most other properties. I'm quite surprised the openapi.yml linter doesn't flag missing descriptions.

@fdr @geemus do you think we should have the openapi.yml linter check for missing property descriptions?